extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -108,6 +108,7 @@ export const invalidDataSchemaProvider = localize('invalidDataSchemaProvider', "
 export const invalidDatabaseReference = localize('invalidDatabaseReference', "Invalid database reference in .sqlproj file");
 export const databaseSelectionRequired = localize('databaseSelectionRequired', "Database selection is required to import a project");
 export const databaseReferenceAlreadyExists = localize('databaseReferenceAlreadyExists', "A reference to this database already exists in this project");
+export const ousiderFolderPath = localize('outsideFolderPath', "Files are added to project file with absolute paths outside of projects drive. please make the path relative and add files from the same drive");
```

Comment:
A quick nit-read :)
* "adding to project with..." (avoiding "file" twice so close)
* project's drive (apostrophe)
* Please (capitalized)
* drive. (period)